main/src/ca/uwaterloo/flix/language/phase/Synthesize.scala

```diff
@@ -1356,17 +1356,17 @@ object Synthesize extends Phase[Root, Root] {
       * Returns `true` if `tpe` is a record type.
       */
     def isRecord(tpe: Type): Boolean = tpe.typeConstructor match {
```

Comment:
I wonder: Could we not just ask if the kind of `tpe` is Record? I guess that might not work until kinds are fully implemented?

Reply:
I think we could, but I prefer to hold off until we have implemented kinds.